core/workspace.js

```diff
@@ -162,19 +154,37 @@ Blockly.Workspace.prototype.MAX_UNDO = 1024;
 Blockly.Workspace.prototype.connectionDBList = null;
 
 /**
- * Adds cursor for keyboard navigation.
- * @return {!Blockly.Cursor} Cursor for keyboard navigation.
+ * Sets the cursor for keyboard navigation.
+ * @param {Blockly.Cursor} cursor The cursor used to navigate around the blockly
+ *     ast for keyboard navigation.
+ */
+Blockly.Workspace.prototype.setCursor = function(cursor) {
+  this.cursor = cursor;
```

Comment:
is this.cursor defined anywhere? Might want to keep it in the constructor and set it to null, so you can keep its annotation.

Reply:
Also, should it be private?

Reply:
done